parachains/runtimes/assets/asset-hub-kusama/src/lib.rs

```diff
@@ -1078,7 +1179,7 @@ impl_runtime_apis! {
 		fn dispatch_benchmark(
 			config: frame_benchmarking::BenchmarkConfig
 		) -> Result<Vec<frame_benchmarking::BenchmarkBatch>, sp_runtime::RuntimeString> {
-			use frame_benchmarking::{Benchmarking, BenchmarkBatch, BenchmarkError,
+			use frame_benchmarking::{Benchmarking, BenchmarkBatch, TrackedStorageKey, BenchmarkError,
```

Comment:
Seems to be imported twice:
```suggestion
			use frame_benchmarking::{Benchmarking, BenchmarkBatch, BenchmarkError,
```